README.md

```diff
@@ -3,80 +3,101 @@
-   cred attribution can't be a black-box. SourceCred will provide tools that
-   make it easy to dive into the cred attribution, and see exactly why
-   contributions were valued the way they were.
+We think projects will want to have an accumulating token that represents
+someone's aggregated support for a project over time. This token is called
+grain. Projects that turn on grain can "mint" new grain every week and issue it
+to contributors in proportion to their cred.
 
-2. **Community-controlled**
+In the future, it will be possible to turn a project's grain into a
+transferrable cryptoasset (likely an [ERC20 token] on [Ethereum]). At that
+point, grain can be seen as a "proof of patronage" of the project. For example,
+people who have SourceCred grain have either directly contributed to SourceCred
+or bought the grain from someone who did, and thus are "patrons" of the
+project.
 
-   At the end of the day, the community of collaborators in a project will know
-   best which contributions were important and deserve the most cred. No
-   algorithm will do that perfectly on its own. To that end, we'll empower the
-   community to modify the cred attribution, by adding human knowledge into the
-   cred graph.
+[ERC20 token]: https://cointelegraph.com/explained/erc-20-tokens-explained
+[Ethereum]: https://ethereum.org/
 
-3. **Forkable**
+#### Current Status
 
-   Disputes about cred attribution are inevitable. Maybe a project you care
-   about has a selfish maintainer who wants all the cred for themself :(. Not
-   to worry—all of the cred data will be stored with the project, so you are
-   empowered to solve cred disputes by forking the project.
+As of July 2018, it's still early days for SourceCred! So far, we have a prototype
+cred explorer that can download Git and GitHub data for a repository, create a
+contribution graph, and run PageRank on it to produce a preliminary cred attribution.
 
-## Roadmap
-SourceCred is currently in a very early stage. We are working full-time to
-develop a MVP, which will have the following basic features:
+If you'd like to try it out, you can run a local copy of SourceCred using the following
+commands. You need to have [node] and [yarn] installed first. You also need to get a
+[GitHub API access token].
 
-- **Create**: The *GitHub Plugin* populates a project's GitHub data into a
-  Contribution Graph. SourceCred uses this seed data to produce an initial,
-  approximate cred attribution.
+[node]: https://nodejs.org/en/
+[yarn]: https://yarnpkg.com/lang/en/
+[GitHub API access token]: https://github.com/settings/tokens
 
-- **Read**: The *SourceCred Explorer* enables users to examine the cred
-  attribution, and all of the contributions in the graph. This reveals why the
-  algorithm behaved the way that it did.
+```
+git clone https://github.com/sourcecred/sourcecred.git
+cd sourcecred
+yarn install
+yarn backend
+node bin/sourcecred.js load $REPO_OWNER $REPO_NAME --github-token=$GH_TOKEN
```

Comment:
Suggest replacing `$REPO_OWNER` with `REPO_OWNER`, because:
 1. it’s a standard form in documentation (e.g., `man cp`), and
 2. in the event that someone blindly copies the literal text, the
    placeholders won’t expand to the empty string and be omitted from
    the command entirely.

Reply:
Done.